plugins/automap/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@flatfile/plugin-automap",
+  "version": "0.0.1",
```

Comment:
❔ Should we keep the version like this for a pre 1.0 release?

Reply:
yes